packages/syntax-core/src/Button/Button.module.css

```diff
@@ -130,23 +130,3 @@
   stroke-dashoffset: 0;
   transform-origin: center;
 }
-
-/** TODO Change with `Typography` component when available */
-.buttonText {
-  color: inherit;
-  font-family: -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto,
-    Helvetica, Arial, sans-serif;
-  font-weight: 500;
```

Comment:
@somethiiing looks like we lost the boldness of the button text - could you work with @ajmooo to see if we can use `bold` or `semiBold` - if not, feel free to add a `span` with a classNames inside of `Typography` which has `font-weight: 500;`